packages/eslint-plugin-jest/src/rules/types.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ */
+export type EslintContext = {
+  report: (message: any) => void
```

Comment:
I've re-written the type definitions to be more accurate (although not complete) to the actual ast types.